logger.js

```diff
@@ -54,17 +61,27 @@ function pinoLogger (opts, stream) {
       return
     }
 
-    log[level]({
-      res: this,
-      responseTime: responseTime
-    }, 'request completed')
+    var shouldLogSuccess = true
+    if (this._currentUrl && autoLoggingOptions && autoLoggingOptions.ignorePaths && autoLoggingOptions.ignorePaths.length) {
+      shouldLogSuccess = autoLoggingOptions.ignorePaths.find(x => this._currentUrl.match(x)) == null
```

Comment:
Would you mind to:

1. extract `ignorePaths` into its own variable
2. remove the use of `find()` and use a for(;;) loop instead? `find()` is significantly slower.

Reply:
fixed